INSTALL.md

```diff
@@ -26,6 +26,7 @@ targets.
  - Modern C++ compiler that supports `-std=c++14`
    - clang++ 3.5 or later _or_
    - g++-5 or later
+ - CMake (for build only)
  - Python 2.x (for build only)
  - [Node.js](https://nodejs.org/) (for build only)
  - [`pkg-config`](https://wiki.freedesktop.org/www/Software/pkg-config/) (for build only)
```

Comment:
We are not using `find_package` in CMake. However, we are using it indirectly because we are invoking `mason` (which could call `pkg-config` based on the `script.sh`) when there is no `mason.ini` file yet in the package. As we move on and more of the packages we use contain a `mason.ini` in the precompiled tarball download, we can remove the on-demand `mason.ini` creation, which removes the `pkg-config` requirement for Mapbox GL Native.

For what it's worth, we only require Python for creating the shader files and version headers, but we can easily move this to node.js, which we require anyway because of our use of `npm`.